content/en/docs/reference/setup-tools/kubeadm/kubeadm-init.md

```diff
@@ -220,11 +220,11 @@ kubeadm config images pull
 You can pass `--config` to the above commands with a [kubeadm configuration file](#config-file)
 to control the `kubernetesVersion` and `imageRepository` fields.
 
-All default `k8s.gcr.io` images that kubeadm requires support multiple architectures.
+All default `registry.k8s.io` images that kubeadm requires support multiple architectures.
 
 ### Using custom images {#custom-images}
 
-By default, kubeadm pulls images from `k8s.gcr.io`. If the
+By default, kubeadm pulls images from `registry.k8s.io`. If the
```

Comment:
Is this right?
```suggestion
For Kubernetes v1.24.0 through to v1.24.7, kubeadm uses images from `k8s.gcr.io`.
The most recent v1.24 patch releases (v1.24.8 and later) switch so that the kubeadm tool
defaults to pulling container images from `registry.k8s.io`.
See [registry.k8s.io: faster, cheaper and Generally Available](/blog/2022/11/28/registry-k8s-io-faster-cheaper-ga/)
for more details on this change.
If the
```

Reply:
the default change was introduced in 1.24.9.

For Kubernetes v1.24.0 through to 1.24.8

Reply:
Could work, feel free to propose it! Use a `note` if you do.